docker-compose.yml

```diff
@@ -3,8 +3,8 @@ services:
   pg:
     image: postgres:9.3
 
-  possum:
-    image: registry.tld/possum:0.1.0-stable
+  conjur:
+    image: registry.tld/conjur:0.1.0-stable
```

Comment:
this should be the public image `cyberark/conjur`, `latest` is stable

Reply:
done.